### PR TITLE
Additional time layout

### PR DESCRIPTION
**Pull request by Tang8330**

### CODE REVIEW

File: lib/typing/ext/variables.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ var supportedDateTimeLayouts = []string{
 	RFC3339Nanosecond,
 	time.RFC3339Nano,
```
Comment (danafallon) [inline lib/typing/ext/variables.go:17]: Just noticed that this one is in here twice

Comment (Tang8330) [reply lib/typing/ext/variables.go:17]: Nice!

Comment (Tang8330) [reply lib/typing/ext/variables.go:17]: I'm gonna write a test to make sure it's only used once

Comment (Tang8330) [reply lib/typing/ext/variables.go]: :+1:

**Review by danafallon — APPROVED** [review]

**Pull request merged**